salt/orch/update-etc-hosts.sls

```diff
@@ -1,13 +1,18 @@
+{% if salt['mine.get']('roles:kube-(master|minion)', 'caasp_fqdn', expr_form='grain_pcre').values()|length > 0 %}
 update_pillar:
   salt.function:
-    - tgt: '*'
+    - tgt: 'roles:kube-(master|minion)'
```

Comment:
This should also target `G@bootstrap_complete:true` as the act of getting info for the pillar and grains can lock the node as well (just for a shorter period of time than running a state)